packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -15,16 +17,11 @@ export interface ILocalFileDestinationProviderOptions {
     key?: string;
   };
 
-  // Compressions
+  // Compression
   compression: {
     enabled: boolean;
   };
 
-  // Archive
```

Comment:
Is it possible to leave this even if we don't provide it in the CLI? It could be very useful for debugging if we could export directly to files sometimes. But not if it's going to complicate things.

Reply:
Ok, I agree

Reply:
Agreed, too much work for nothing that's currently on the roadmap.